CONTRIBUTING.md

```diff
@@ -65,23 +65,5 @@ If you have already authored a commit that is missing the signed-off, you can am
 
 [DCO]: /DCO
 
-## Common tasks
 
-### Linting Protobuf Definitions
 
-All [Protobuf] code is managed using [buf].
-You can lint the definitions by executing:
-
-```sh
-buf build && buf lint
-```
```

Comment:
Why was this removed? Developers should still be able to generate and lint locally as they change the gRPC APIs

Reply:
Disclaimer: i'm not an expert on Buf (yet)

According to the note

> `Generate` builds and then produces actual source code according to `buf.gen.yaml`.

I added `buf-generate` as part of the pre-commit hooks, so wouldn't this be the same outcome?

On a related note: I'm surprised that this repo doesn't contain the Go files, why is that and should we add them?

Reply:
Ok 👍